### DISCUSSION TITLE
ATSAS 4.0 operating systems

### DISCUSSION THREAD

**Discussion by sofiakap** [biosaxs-com/atsas-community#1959] [ATSAS package in general]
Hello,
it is mentioned on the BioSAXS website that ATSAS is supported for the following operating systems: Windows 11
MacOS (11.X Big Sur, 12.X Monterey, 13.X Ventura, 14.X Sonoma). Does this mean that it will not work properly on Windows 10 and macOS Catalina, version 10.15.7? If not, how to deal with this besides upgrading? thank you.

**Comment by franke-biosaxs** [comment]
The Windows packages have been built on Windows 11 but might work on Windows 10. You are free to try, but no guarantees. One user reported "lagging" when run on Windows 10.

On macOS 10.15: that is past its intended end-of-life by two years now. There's nothing we can do to support this at this stage.
